context/context.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
Don't follow the distribution pattern. We'll probably rip that out in favor the containerd/swarmkit approach.